common-types/src/main/scala/com/improving/app/common/errors/Validation.scala

```diff
@@ -41,7 +41,7 @@ object Validation {
 
   def listHasLength[T]: String => Validator[Seq[T]] = fieldName => { list =>
     if (list.isEmpty) {
-      Some(ValidationError("List " + fieldName + "has no length"))
+      Some(ValidationError("List " + fieldName + " has no length"))
```

Comment:
is empty without a length or is the length zero?

Reply:
Length zero. It could only be 'without a length' if it were wrapped in `Option.None`, but the type on `listHasLength` has no `Option`

Reply:
Ah I see what you're saying now - no necessarily the clearest statement. Can change for next PR.